pkg/command/cmd.go

```diff
@@ -59,16 +58,16 @@ func RunWithTimeout(args string, timeout int) (string, error) {
 	return stdout.String(), nil
 }
 
-func Run(args string) (string, error) {
+func Run(args string, log logr.Logger) (string, error) {
 	timeoutString, found := os.LookupEnv("NNF_COMMAND_TIMEOUT_SECONDS")
```

Comment:
Was this never set before?

Reply:
The timeout? This Run() function is new with my code. It wasn't set until now. There's a separate timeout env variable for commands run from nnf-ec